source/reference/methods.txt

```diff
@@ -444,6 +444,72 @@ Database Methods
 
      - Provides access to the specified database.
 
+   * - :method:`db.logout()`
+
+     - Ends an authenticated session.
+
    * - :method:`db.runCommand()`
 
      - Runs a :manual:`database command </reference/command>`.
+
+User Management Methods
+-----------------------
+
+.. important::
+
+   The :method:`passwordPrompt()` method is currently not
+   supported in ``mongosh``. This limitation affects behavior
+   for the following methods:
```

Comment:
I'm guessing that "affects behavior" means that the user has to enter the password as a parameter instead of using the password prompt -- can we make that clearer here? Otherwise I think it's hard to really draw any information out of this admonition unless you're already very familiar with `passwordPrompt()`.

Reply:
Good call! Updated with more context. Thanks Nate